src/main/resources/database/changes/release-10.49.2/us002a-survey-ref-to-uuid.sql

```diff
@@ -0,0 +1,12 @@
+
+ALTER TABLE collectionexercise.collectionexercise ADD COLUMN survey_uuid UUID;
+
+UPDATE collectionexercise.collectionexercise AS ce
+SET survey_uuid = subquery.id
+FROM (
+    SELECT surveypk, id
+    FROM collectionexercise.survey
+) AS subquery
+WHERE ce.surveyfk = subquery.surveypk;
+
+ALTER TABLE collectionexercise.collectionexercise DROP COLUMN surveyfk;
```

Comment:
Do we need to drop the fk constraint or will that be cascaded?

Reply:
I am pretty sure it cascades - also the script worked no problem on the docker postgres